content/doc/upgrade-guide/2.73.adoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: documentation
+title:  Jenkins Upgrade Guide
+notitle: true
+---
+
+== Upgrading to Jenkins LTS 2.73.x
+
+Each section covers the upgrade from the previous LTS release, the section on 2.73.1 covers the upgrade from 2.60.3.
+
+=== Upgrading to Jenkins LTS 2.60.1
+
+==== Groovy 2.4.11 Upgrade
+
+link:https://issues.jenkins-ci.org/browse/JENKINS-43197[JENKINS-43197]
+
+Groovy was updated from 2.4.8 to 2.4.11.
+This may resolve existing Groovy-related memory problems, as 2.4.9 contains the fix for link:https://issues.apache.org/jira/browse/GROOVY-8067[GROOVY-8067].
+It may no longer be necessary to apply the `groovy.use.classvalue` workaround.
```

Comment:
@jglick Can we provide some clear guidance here, or are we really not sure what users should be doing?